test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -156,8 +156,8 @@ def timeout(val):
     """
     Decorator to set timeout value of test case
     """
-    if val < DEFAULT_TESTCASE_TIMEOUT:
-        val = DEFAULT_TESTCASE_TIMEOUT
+    if val < MINIMUM_TESTCASE_TIMEOUT:
+        val = MINIMUM_TESTCASE_TIMEOUT
```

Comment:
this is where you might wanna add a warning message

Reply:
I will add warning message here, but timeout decorator evaluate initially in test suite it will comes initially in testsuite not at the test level.

Reply:
@dinesh1306 thanks for the info. I suggest still adding a generic message mentioning that some of the tests are updated to use minimum_test_timeout

Reply:
@anamikau , I have added message.